Example/SBTUITestTunnel_Tests/MiscellaneousTests.swift

```diff
@@ -181,6 +181,30 @@ class MiscellaneousTests: XCTestCase {
                 
         XCTAssert(app.staticTexts["80"].isHittable)
     }
+
+    func testCollectionViewScrollingVertical() {
+        app.launchTunnel()
+
+        app.cells["showExtensionCollectionViewVertical"].tap()
+
+        XCTAssertFalse(app.staticTexts["50"].isHittable)
+
+        app.scrollTableView(withIdentifier: "collection", toElementWithIdentifier: "50", animated: true)
```

Comment:
Could you add tests that verify the scroll to indexPath `scrollTableView(withIdentifier:toRow:animated:`? Would it make sense to rename `scrollTableView(withIdentifier:toRow:animated:` -> `scrollTableView(withIdentifier:toIndex:animated:`

Reply:
I can add more tests, no problem 👍 

About the renaming, do you mean create an overload with the a signature and deprecation of the current one?

Reply:
I would remove the current one releasing a new major.

Reply:
OK perfect, I'll open a separate PR for the naming improving